### DISCUSSION TITLE
vaccine Hesitancy by Gender, age group and other reasons

### DISCUSSION THREAD

**Discussion by studentuwe** [owid/covid-19-data#2503] [Q&A] [answered]
Hi, I am a university student researching on Reasons of Vaccine Hesitancy in U.K and Nigeria. It would be greatly appreciated if you assist in getting data.

**Comment by lucasrodes** [comment, marked as answer]
Hi @studentuwe,
Sorry for the delay in our response.

Due to our limited resources, we currently don't report vaccination hesitancy by gender and don't plan to do so in the short term.

We have data on attitudes on vaccination, which you can check [here](https://ourworldindata.org/attitudes-to-covid-vaccinations). However, this data is aggregated and does not include Nigeria.
